Core/include/Acts/EventData/Measurement.hpp

```diff
@@ -236,7 +238,7 @@ class Measurement {
   /// @return vector with the residual parameter values (in valid range)
   ///
   /// @sa ParameterSet::residual
-  ParVector_t residual(const TrackParameters& trackPars) const {
+  ParVector_t residual(const BoundParameters& trackPars) const {
```

Comment:
Just to be sure I understand your comment in the PR, BoundParameters will be change to BoundTrackParameters in a second PR right ?

Reply:
I am also not completely sure why you replace the SingleTrackParameters by a SingleBoundTrackParameters. But I am not necessarily an expert on the subject.

Reply:
Yes, the `BoundParameters` will be changed in a second PR.

The `TrackParameter` typedef was removed as it is just a base class for `BoundParameters` and `CurvinlinearParameters`.